en/docs/insights/choreo-insights-api.md

```diff
@@ -1841,6 +2042,150 @@ The granularity value for which data is retrieved.
 </tbody>
 </table>
 
+### CustomReportGroupBy
+
+Represents usage data overtime for the selected paramters for a certain groupByValue
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>groupByValue</strong></td>
+<td valign="top"><a href="#id">ID</a>!</td>
+<td>
+
+Distinct group by value used as an ID
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>usage</strong></td>
+<td valign="top">[<a href="#customreportusage">CustomReportUsage</a>]</td>
+<td>
+
+A list containing the usage values. Returns an empty array if no data is available.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### CustomReportOvertime
+
+Represents usage data overtime for the selected paramters
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>usage</strong></td>
+<td valign="top">[<a href="#customreportusage">CustomReportUsage</a>]</td>
+<td>
+
+A list containing the usage values. Returns an empty array if no data is available.
```

Comment:
Shall we change the second sentence to "This returns an empty array if no data is available."

Reply:
ack